tests/network-tests/src/cli/utils.ts

```diff
@@ -18,7 +18,7 @@ export class TmpFileManager {
     )
     mkdirSync(this.tmpDataDir, { recursive: true })
     nodeCleanup(() => {
-      rmSync(this.tmpDataDir, { recursive: true, force: true })
+      //      rmSync(this.tmpDataDir, { recursive: true, force: true })
```

Comment:
Is there a reason for commenting it out, or was it a mistake?

Reply:
This was commented out for debug purposes. Fixed. On Olympia there is a better mechanism built in to debug CLI results.